projects/api/src/lib/models/viewer-response.model.ts

```diff
@@ -2,6 +2,7 @@ import { ComponentModel } from './component.model';
 import { ViewerStylingModel } from './viewer-styling.model';
 import { I18nSettingsModel } from './i18n-settings.model';
 import { UiSettingsModel } from './ui-settings.model';
+import { FilterGroupModel } from '../../../../core/src/lib/filter/models/filter-group.model';
```

Comment:
This introduces a circular dependency where `api` starts to depend on `core` (but there already is a peerDependency from `core` on `api`: https://github.com/Tailormap/tailormap-viewer/blob/10b8b70743a419190e4f049422dde0953d7c8238/projects/core/package.json#L27